server/shops.lua

```diff
@@ -6,7 +6,7 @@ local function AddItems(stash, Items)
     local items = {}
 
     for k, v in pairs(Items) do
-        local itemInfo = QBCore.Shared.Items[k:lower()]
+        local itemInfo = QBCore.Shared.Items[k.item]
```

Comment:
I believe the changes you made here need to be reverted aswell, k should be the name of the item and v should be the quantity. Neither value should be a table